kakine.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency 'fog'
+  spec.add_dependency "yao", "~> 0.1.0"
```

Comment:
yao's current version is `0.2.7`, so I hope here to use `~> 0.2.0` :exclamation: